fetch/fragments/subresource-redirect.html

```diff
@@ -0,0 +1,7 @@
+<p><img src=/images/green.svg>
```

Comment:
I guess this isn't ready for review yet since this isn't used, will hold off until poked.

Reply:
Yeah, I was thinking of using a reference test for this, but I first need to determine if everyone is okay with passing along fragments all the time.